auraed/src/runtime/pod_service/pod_service.rs

```diff
@@ -31,27 +31,33 @@
 #![allow(unused)]
 #![allow(clippy::module_inception)]
 
+use anyhow::{Context, Result};
```

Comment:
I'm not spotting where this file is even using `anyhow`, but we should refactor it out in a future PR